.github/workflows/_sandbox.yaml

```diff
@@ -2,10 +2,27 @@ name: "~Sandbox"
 
 on:
   workflow_dispatch:
+  workflow_run:
```

Comment:
Don't forget to reverse it, when done with this PR

Reply:
Done